ImageRater/MainWindow.xaml.cs

```diff
@@ -1,18 +1,28 @@
 ﻿namespace ImageRater;
 
+using System.IO;
 using System.Windows;
+using System.Windows.Forms;
 using ImageRater.ViewModels;
 
 /// <summary>
 /// Interaction logic for MainWindow.xaml
 /// </summary>
 public partial class MainWindow : Window
 {
-    public ImageRaterViewModel ImageRater { get; set; }
+    public ImageRaterViewModel? ImageRater { get; set; }
     public MainWindow()
     {
-        this.ImageRater = new("C:\\workingDirectory\\ForschungsPraktikum\\D600\\screenshots\\screenshots\\screen_all_in_one_folder");
-        this.DataContext = this.ImageRater;
+        using (var dialog = new FolderBrowserDialog())
```

Comment:
Can you give the dialog a descriptive title?
Maybe something like "Please select your image input folder".